sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -8,112 +8,313 @@ The authentication events trigger for Azure Functions allows you to implement a
+3. Use the following JSON body that imitates the request Microsoft Entra ID sends to your REST API.
+
+```json
+{
+    "type": "microsoft.graph.authenticationEvent.tokenIssuanceStart",
+    "source": "/tenants/aaaabbbb-0000-cccc-1111-dddd2222eeee/applications/00001111-aaaa-2222-bbbb-3333cccc4444",
+    "data": {
+        "@odata.type": "microsoft.graph.onTokenIssuanceStartCalloutData",
+        "tenantId": "aaaabbbb-0000-cccc-1111-dddd2222eeee",
+        "authenticationEventListenerId": "11112222-bbbb-3333-cccc-4444dddd5555",
+        "customAuthenticationExtensionId": "22223333-cccc-4444-dddd-5555eeee6666",
+        "authenticationContext": {
+            "correlationId": "aaaa0000-bb11-2222-33cc-444444dddddd",
+            "client": {
+                "ip": "127.0.0.1",
+                "locale": "en-us",
+                "market": "en-us"
+            },
+            "protocol": "OAUTH2.0",
+            "clientServicePrincipal": {
+                "id": "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb",
+                "appId": "00001111-aaaa-2222-bbbb-3333cccc4444",
+                "appDisplayName": "My Test application",
+                "displayName": "My Test application"
+            },
+            "resourceServicePrincipal": {
+                "id": "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb",
+                "appId": "00001111-aaaa-2222-bbbb-3333cccc4444",
+                "appDisplayName": "My Test application",
+                "displayName": "My Test application"
+            },
+            "user": {
+                "companyName": "Casey Jensen",
+                "createdDateTime": "2023-08-16T00:00:00Z",
+                "displayName": "Casey Jensen",
+                "givenName": "Casey",
+                "id": "00aa00aa-bb11-cc22-dd33-44ee44ee44ee",
+                "mail": "casey@contoso.com",
+                "onPremisesSamAccountName": "Casey Jensen",
+                "onPremisesSecurityIdentifier": "<Enter Security Identifier>",
+                "onPremisesUserPrincipalName": "Casey Jensen",
+                "preferredLanguage": "en-us",
+                "surname": "Jensen",
+                "userPrincipalName": "casey@contoso.com",
+                "userType": "Member"
+            }
+        }
+    }
+}
+```
+
+4. Select **Send**, and you should receive a JSON response similar to the following:
+
+```json
+{
+    "data": {
+        "@odata.type": "microsoft.graph.onTokenIssuanceStartResponseData",
+        "actions": [
+            {
+                "@odata.type": "microsoft.graph.tokenIssuanceStart.provideClaimsForToken",
+                "claims": {
+                    "customClaim1": "customClaimValue1",
+                    "customClaim2": [
+                        "customClaimString1",
+                        "customClaimString2" 
+                    ]
+                }
+            }
+        ]
+    }
+}
+```
+
+### Deploy the function and publish to Azure 
```

Comment:
We do not need to give instruction on how to publish to Azure.   That is beyond the scope of "how does a Functions developer use your extension"

Reply:
removed step by step instructions